reladomographql/src/test/java/com/gs/reladomo/graphql/SchemaProviderTest.java

```diff
@@ -0,0 +1,206 @@
+                        "{\"orderId\":4,\"description\":\"Fourth order, different user\",\"items\":[]}]}\n");
+    }
+
+    @Test
+    public void findManyRelationshipTraversal() throws Exception {
+        assertQuery(
+                "{ orderItems (findMany: {order: { orderId: {in: [2,3,4,10]}} }) { id state order { description orderId } } }",
+                "{\"orderItems\":[" +
+                        "{\"id\":2,\"state\":\"InProgress\",\"order\":{\"description\":\"Second order\",\"orderId\":2}}," +
+                        "{\"id\":3,\"state\":\"InProgress\",\"order\":{\"description\":\"Second order\",\"orderId\":2}}," +
+                        "{\"id\":4,\"state\":\"InProgress\",\"order\":{\"description\":\"Second order\",\"orderId\":2}}" +
+                        "]}");
+    }
+
+    @Test
+    public void metadata() throws Exception {
+        assertQuery(
+                "{ productById ( productId: 1 ) { productId, productCode } }",
+                "{\"productById\":{\"productId\":1,\"productCode\":\"AA\"}}");
+    }
+
+    @Test
+    public void reladomoFinder() throws Exception {
+        assertQuery("{ orders (findMany: {orderId: {in: [2,3,4,10]} or: {state: {startsWith: \"In\"}}}) { orderId description items { state } } }",
+                "{\"orders\":[" +
+                        "{\"orderId\":3,\"description\":\"Third order\",\"items\":[]}," +
+                        "{\"orderId\":2,\"description\":\"Second order\",\"items\":[{\"state\":\"InProgress\"},{\"state\":\"InProgress\"},{\"state\":\"InProgress\"}]}," +
+                        "{\"orderId\":4,\"description\":\"Fourth order, different user\",\"items\":[]}," +
+                        "{\"orderId\":1,\"description\":\"First order\",\"items\":[{\"state\":\"InProgress\"}]}]}\n");
+    }
+
+    @Test
+    public void orderByAndLimit() throws Exception {
+        assertQuery(
+                "{ orders (findMany: {orderId: {notIn: [4,10]}} order_by: {orderId: desc}  limit: 2) { orderId description} }",
+                "{\"orders\":[" +
+                        "{\"orderId\":3,\"description\":\"Third order\"}," +
+                        "{\"orderId\":2,\"description\":\"Second order\"}," +
+                        "{\"orderId\":1,\"description\":\"First order\"}" +
+                        "]}");
+    }
+
+    @Test
+    public void multipleQueries() throws Exception {
+        assertQuery(
+                "{ productById(productId: 4) {  productId  productCode  } orderById(orderId: 3) { orderId description } }",
+                "{\"productById\":{\"productId\":4,\"productCode\":\"AB\"}," +
+                        "\"orderById\":{\"orderId\":3,\"description\":\"Third order\"}}\n");
+    }
+
+    @Test
+    public void aggregateSum() throws Exception {
+        assertQuery(
+                "{ testBalanceNoAcmap_aggregate (filter: {productId: {in: [1, 2, 3, 4, 5]} }) { accountId sumOfQuantity }} \n",
```

Comment:
I'm curious about this design decision as well. How did you decide to make aggregate part of the field name, rather than part of the parameter object?